application/src/main/java/org/opentripplanner/routing/algorithm/mapping/RouteRequestToFilterChainMapper.java

```diff
@@ -46,6 +48,11 @@ public static ItineraryListFilterChain createFilterChain(
       builder = builder.withPagingDeduplicationFilter(request.pageCursor().itineraryPageCut());
     }
 
+    // The page cursor has best street only cost information only in certain cases.
```

Comment:
You could document what those certain cases are.

Reply:
I documented this